_posts/2017-02-17-holacracy-and-purpose.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Holacracy and purpose
+---
+
+{{page.title}}
+==============
+
+**In Holacracy, purpose is seen as the core concept around which everything else revolves. But what makes purpose so vital for a self-managing organization, and why should you care?**
+
+## A thought experiment
+
+Let's assume you are the managing director of a software company with three departments: Sales, Engineering, and Recruitment.
+
+One day, you get contrasting reports from two of your departments. Sales figures are better than ever before. Engineering, however, reports high work loads and not being able to deliver software on time.
+
+As a strategic direction, you decide engineering capacity should be increased. You ask the Recruitment department to take up a project to hire two new software engineers. As a first action, they should write up a job description and send it to you for final approval.
+
+## Same situation, different approach
+
+Now, let's do this same experiment for a self-managing organization running on Holacracy. I assume familiarity with Holacracy's basic concepts. Otherwise, consider watching [this animation video](https://www.youtube.com/watch?v=MUHfVoQUj54) or [this talk by Brian Robertson](https://www.youtube.com/watch?v=tJxfJGo-vkI) to get a quick overview.
+
+For the context of this article, we'll focus on the concept of _Lead Link_. In the Holacracy operating system, there is no such thing as a "managing director" or other "manager" or "leader" job titles. Instead, work is divided over circles and roles. Each circle has a purpose and a special role called Lead Link. One of the Lead Link's responsibilities is to ensure the circle is working towards its purpose.
```

Comment:
Je bedoelt én "assume familiarity" overslaan én de alinea's over Lead Link en Anchor Circle? Ik had verwacht dat je zou zeggen: kies één van de twee... Ik denk dat ik je niet goed begrijp dan, of wel?